pkg/acsengine/testdata/vnet/kubernetesvnet.json

```diff
@@ -49,6 +49,11 @@
       "clientPrivateKey": "clientPrivateKey",
       "kubeConfigCertificate": "kubeConfigCertificate",
       "kubeConfigPrivateKey": "kubeConfigPrivateKey"
+    },
+    "kubeNetworkConfig": {
+      "kubeDnsServiceIp": "172.17.1.10",
```

Comment:
How is the backward compatibility handled here ? I am not saying we need backward compatibility or not. Just wondering in your change, how this get treated ?